.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: [3.9, 3.8, 3.7, 3.6]
```

Does it work on the 3.10 release candidate? It's due out in a couple of weeks, it's worth testing.

```suggestion
        python-version: ["3.10-dev", "3.9", "3.8", "3.7", "3.6"]
```